front50-web/src/test/groovy/com/netflix/spinnaker/front50/controllers/AuthorizationSupportSpec.groovy

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.front50.controllers
+
+import com.netflix.spinnaker.fiat.shared.FiatPermissionEvaluator
+import com.netflix.spinnaker.front50.model.pipeline.Pipeline
+import spock.lang.Specification
+import spock.lang.Subject
+import spock.lang.Unroll
+
+public class AuthorizationSupportSpec extends Specification {
+
+  @Unroll
+  def "should validate run as user access"() {
+    setup:
+    FiatPermissionEvaluator evaluator = Mock(FiatPermissionEvaluator)
+    @Subject AuthorizationSupport authorizationSupport = new AuthorizationSupport(permissionEvaluator: evaluator)
+
+    when:
+    Pipeline p = new Pipeline()
+    authorizationSupport.hasRunAsUserPermission(p)
```

Comment:
Are you expecting that `authorizationSupport.hasRunAsUserPermission(p)` asserts true?

Normally that should be done in `then:`.

```
when:
emptyPipelineHasRunAsUserPermission = hasRunAsUserPermission(new Pipeline()) 

then:
emptyPipelineHasRunAsUserPermission == true
```

or

```
expect:
authorizationSupport.hasRunAsUserPermission(new Pipeline()) == true
authorizationSupport.hasRunAsUserPermission(new Pipeline(triggers: [])) == true
```

Am I miss interpreting something here?